### PR TITLE
Minions with basic Orders

### PR DESCRIPTION
**Pull request by klawir**

### CODE REVIEW

**Review by NamespaceV — APPROVED** [review]

File: Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs [hunk 1]
```diff
@@ -154,8 +160,6 @@ private void Start()
 
         private void Update()
         {
-            _hasAnimator = TryGetComponent(out _animator);
```
Comment (NamespaceV) [inline Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs]: don't delete this in PR this large

File: Assets/Prefab/Capsule.prefab
Comment (NamespaceV) [inline Assets/Prefab/Capsule.prefab:1]: rename to `Minion.prefab`

File: Assets/Scripts/Runtime/Npc/Minion.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+{
+    public class Minion : Creature
+    {
+        [SerializeField] private UnityEngine.AI.NavMeshAgent localNavMeshAgent;
+        [SerializeField] private float minimumDistanceToStartFollowTheCharacterPlayer;
+
+        public System.Action<Minion> OnFishedOrder;
+        [SerializeField] private Vector3 m_newPosition;
+        private bool _isGoingAlready;
+
```
Comment (NamespaceV) [inline Assets/Scripts/Runtime/Npc/Minion.cs]: Unify to a single private field naming style. Here there are 3 styles:
- `camelCase`
- `_underscoreForPrivate`
- `m_memberPrefixWithUnderscore`

Unity style guide allows all, but mixing all three is confusing.
https://unity.com/how-to/naming-and-code-style-tips-c-scripting-unity#fields-and-variables

Microsoft recommends  `_fieldName`
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions
only suggests using `s_fieldName` for static fields

**Review by NamespaceV — APPROVED** [review]

File: Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs (hunk 1 above)
Comment (NamespaceV) [reply Assets/StarterAssets/ThirdPersonController/Scripts/ThirdPersonController.cs]: if think this should stay for now
if we want to delete this line do it in separate pr (with this single line)

**Comment by Maciekxdabu** [comment]
Jak dla mnie wygląda git (nie przeglądałem kodu, ale zobaczyłem jak działa), chyba jedyny mankament dla mnie, że UI'ka z Canvasem się źle skaluje, napis powinien być przypięty do rogu Canvas'u, a nie środka

**Pull request merged**